src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -108,27 +108,39 @@
             </trans-unit>
             <trans-unit id="27">
                 <source>This value is not a valid URL.</source>
-                <target>Värdet är inte en giltig URL.</target>
+                <target>Värdet är inte en giltig webbadress.</target>
+            </trans-unit>
+            <trans-unit id="28">
+                <source>This form should not contain extra fields.</source>
+                <target>Formuläret bör inte innehålla extra fält.</target>
+            </trans-unit>
+            <trans-unit id="29">
+                <source>The uploaded file was too large. Please try to upload a smaller file.</source>
+                <target>Den uppladdade filen var för stor. Försök ladda upp en mindre fil.</target>
+            </trans-unit>
+            <trans-unit id="30">
+                <source>The CSRF token is invalid. Please try to resubmit the form.</source>
+                <target>CSRF-token är ogiltig. Försök skicka formuläret igen.</target>
             </trans-unit>
             <trans-unit id="31">
-                <source>The two values should be equal.</source>
-                <target>De två värdena måste vara lika.</target>
+                <source>This value is not a valid HTML.</source>
+                <target>Värdet är inte giltig HTML.</target>
             </trans-unit>
             <trans-unit id="32">
-                <source>The file is too large. Allowed maximum size is {{ limit }} {{ suffix }}.</source>
-                <target>Filen är för stor. Tillåten maximal storlek är {{ limit }} {{ suffix }}.</target>
+                <source>This value is not a valid International Securities Identification Number (ISIN).</source>
+                <target>Värdet är inte ett giltigt International Securities Identification Number (ISIN).</target>
             </trans-unit>
             <trans-unit id="33">
-                <source>The file is too large.</source>
-                <target>Filen är för stor.</target>
+                <source>This value is not a valid ISBN-10.</source>
+                <target>Värdet är inte ett giltigt ISBN-10.</target>
             </trans-unit>
             <trans-unit id="34">
-                <source>The file could not be uploaded.</source>
-                <target>Filen kunde inte laddas upp.</target>
+                <source>This value is not a valid ISBN-13.</source>
+                <target>Värdet är inte ett giltigt ISBN-13.</target>
             </trans-unit>
             <trans-unit id="35">
-                <source>This value should be a valid number.</source>
-                <target>Värdet ska vara ett giltigt nummer.</target>
+                <source>This value is neither a valid IPv4 address nor a valid IPv6 address.</source>
```

Comment:
Make sure the `trans-unit id` and the source are the same as in validators.en.xlf.